### PR TITLE
ws touch up for Streamed inputs

### PR DESCRIPTION
**Pull request by phderome**
adding handling of `BinaryMessage` and `Streamed`.

### CODE REVIEW

File: akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala [hunk 1]
```diff
@@ -87,69 +88,82 @@ object AkkaHttpAdapter extends FailFastCirceSupport {
...
+            }
+      } yield ()
+
     import akka.http.scaladsl.server.Directives._
 
     get {
       extractUpgradeToWebSocket { upgrade =>
         val (queue, source) = Source.queue[Message](0, OverflowStrategy.fail).preMaterialize()
-        val subscriptions   = runtime.unsafeRun(Ref.make(Map.empty[String, Fiber[Throwable, Unit]]))
+        val subscriptions: Ref[Map[String, Fiber[Throwable, Unit]]] =
```
Comment (ghostdogpr) [inline akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala]: I think there's no need to explicitly put the type here

File: akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala [hunk 2]
```diff
@@ -90,66 +91,80 @@ object AkkaHttpAdapter extends FailFastCirceSupport {
...
-                    case "stop" =>
-                      val id = msg.hcursor.downField("id").success.flatMap(_.value.asString).getOrElse("")
-                      subscriptions
-                        .modify(map => (map.get(id), map - id))
-                        .flatMap(fiber => IO.whenCase(fiber) { case Some(fiber) => fiber.interrupt })
-                  }
-            } yield ()
-            runtime.unsafeRun(io)
+          case TextMessage.Strict(text) => runtime.unsafeRun(processMessage(queue, subscriptions, text))
+          case Streamed(textStream) =>
```
Comment (ghostdogpr) [inline akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala]: This is `BinaryMessage.Streamed`, you wanted `TextMessage.Streamed` no? Better to put it explicitly like above.

File: akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala (hunk 1 above)
Comment (phderome) [reply akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala]: yes, fixed.

File: akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala [hunk 3]
```diff
@@ -90,66 +91,80 @@ object AkkaHttpAdapter extends FailFastCirceSupport {
...
+            textStream
+              .runFold("")(_ + _)
+              .map(message => runtime.unsafeRun(processMessage(queue, subscriptions, message)))
+            ()
+          case bm: BinaryMessage =>
+            bm.dataStream.runWith(Sink.ignore)
+            ()
           case _ => ()
```
Comment (ghostdogpr) [inline akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala]: If you handled all Binary and Text, this line can be removed.

**Comment by phderome** [comment]
We see issues of unwanted duplicated websockets and so duplicated messages sent back to client (Altair for instance). We also see undesirable Akka dead letters going on with this PR, so closing.

**Pull request closed without merging**